src/client/ds3/pool.c

```diff
@@ -148,7 +148,9 @@ ds3_connect(const char *pool, const char *sys, ds3_t **ds3, daos_event_t *ev)
 
 	dfs_disconnect(ds3_tmp->meta_dfs);
```

Comment:
Why not testing also the return code of this function as you have done on line 174 ?

Reply:
it's not reported by coverity ;-)
but yea i'll do that. i just didn't see it.